packages/react-card/src/Card.stories.tsx

```diff
@@ -6,7 +6,7 @@ export { ActionCard } from './stories/CardAction.stories';
 export { GridCard } from './stories/CardGrid.stories';
 
 export default {
-  title: 'Components/Card [ALPHA]',
+  title: 'Preview Components/Card',
```

Comment:
Listed at the bottom of the left nav:

![image](https://user-images.githubusercontent.com/5067638/152237143-5b3c5d30-3a19-41b1-a311-393a4d77ec66.png)